lychee-bin/src/options.rs

```diff
@@ -453,7 +453,7 @@ separated list of accepted status codes. This example will accept 200, 201,
     /// e.g. <https://example.com> or `/path/to/public`
```

Comment:
We think we should also update this help message. `/path/to/public` is no longer recommended and will soon be removed. People should use `root_url` instead. Here's a suggestion:

```
/// Base URL used to resolve relative URLs during link checking
/// Example: "https://example.com"
```